src/utils/browser.ts

```diff
@@ -0,0 +1,36 @@
+import { Browser } from '@uniswap/analytics-events'
```

Comment:
This seems to be copy/pasted now from a few places - should this function be in the base infra package?

Reply:
Good call out. This also occured to me. We can pull this out in a follow up ticket for sure.